src/main/java/org/biouno/unochoice/DynamicReferenceParameter.java

```diff
@@ -131,6 +132,7 @@ public String getChoicesAsStringForUI() {
     // --- descriptor
 
     @Extension
+    @Symbol({"referenceChoice"})
```

Comment:
@imoutsatsos what do you think of these three aliases for pipeline users?

- `activeChoice` -> Active Choices Parameter
- `referenceChoice` -> Active Choices Reactive Reference Parameter
- `reactiveChoice` -> Active Choices Reactive Parameter

I think if we were choosing parameter names today, knowing about this pipelines aliases thing, probably we would give the parameters other names to make it easier to pick up aliases :-) but hindsight is always 20/20 right.